### DISCUSSION TITLE
支持一键部署原版服务器

### DISCUSSION THREAD

**Discussion by Welsonpeaches** [Meloong-Git/PCL#789] [提问] [answered]
支持一键部署原版服务器（或给已搭建的服务器加插件）

**Comment by Pigeon0v0** [comment]
#113

**Comment by WForst-Breeze** [comment, marked as answer]
1. 部署原版服务器请见 #321 ，PCL2 是启动器，而非开服工具，此功能不会制作。
2. 下载和安装插件请见 #108 ，此功能不会制作。
3. 下载原版服务端请见 #113
